tests/unit/lib/test_v3bwfile.py

```diff
@@ -393,3 +394,17 @@ def test_update_progress(datadir, tmpdir):
     assert header.number_consensus_relays == '3'
     assert header.number_eligible_relays == '3'
     assert header.percent_eligible_relays == '100'
+
+
+def test_time_measure_half_network(caplog):
+    header = V3BWHeader(timestamp_l,
+                        file_created=file_created,
+                        generator_started=generator_started,
+                        earliest_bandwidth=earliest_bandwidth)
+    header.number_consensus_relays = '6500'
+    header.number_eligible_relays = '4000'
+    caplog.set_level(logging.INFO)
+    header.add_time_report_half_network()
+    assert header.time_to_report_half_network == '70200'  # 20h
```

Comment:
Hm. Shouldn't the full time of "20h" be 40 hours? Why are we testing for 39 hours?
Is the comment wrong, the logic weird, or something else?

Reply:
ha, yeah, sorry, i rounded 19.5 and didn't commented. Fixup with no rounding.